ssh/proxy.go

```diff
@@ -227,11 +226,11 @@ func (file userFile) read(username string) ([]byte, error) {
 	return ioutil.ReadFile(userSpecFile(username, string(file)))
 }
 
-func (p *ProxyConn) VerifySignature(msg *userAuthRequestMsg, publicKey PublicKey, sig *Signature) (bool, error) {
-	if !isAcceptableAlgo(sig.Format) {
+func (p *ProxyConn) VerifySignature(msg *userAuthRequestMsg, publicKey PublicKey, algo string, sig *Signature) (bool, error) {
+	if !contains(supportedPubKeyAuthAlgos, sig.Format) {
```

Comment:
`isAcceptableAlgo` function is gone. and use algo string for build data signed for auth.